File: News_Service_WebFlux/src/main/java/com/api/AdvancedServiceWebFlux/dto/response/GetNewsOutResponse.java
package com.api.AdvancedServiceWebFlux.dto.response;

import lombok.AllArgsConstructor;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.Set;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode
public class GetNewsOutResponse {

    private String description;

    private Long id;

    private String image;

    private Set<TagResponse> tags;

    private String title;

    private String userId;

    private String username;
}

File: News_Service_Kotlin/src/main/kotlin/com/dunice/advancedServiceKotlin/dto/response/GetNewsOutResponse.kt
package com.dunice.advancedServiceKotlin.dto.response

import lombok.*

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode
class GetNewsOutResponse {
    private val description: String? = null

    private val id: Long? = null

    private val image: String? = null

    private val tags: Set<TagResponse>? = null

    private val title: String? = null

    private val userId: String? = null

    private val username: String? = null
}
